Browser/assertion_engine.py

```diff
@@ -3,32 +3,83 @@
 import re
 
 AssertionOperator = Enum(
-    "AssertionOperator", "NO_ASSERTION == != contains matches < > <= >="
+    "AssertionOperator",
+    "NO_ASSERTION == should_be != should_not_be < > <= >= *= contains matches ^= $="
 )
 
 
-def verify_assertion(value: Any, operator: AssertionOperator, expected, message=""):
-    if operator.name == "==" and value != expected:
-        raise AssertionError(f"{message} `{value}` should be `{expected}`")
-    if operator.name == "!=" and value == expected:
-        raise AssertionError(f"{message} `{value}` should not be `{expected}`")
-    if operator.name == "<" and value >= expected:
-        raise AssertionError(f"{message} `{value}` should be less than `{expected}`")
-    if operator.name == ">" and value <= expected:
-        raise AssertionError(f"{message} `{value}` should be greater than `{expected}`")
-    if operator.name == "<=" and value > expected:
+def _normalize(operator: str):
+    return operator.lower().replace('_', '').replace(' ', '')
+
+
+def verify_assertion(value: Any,
+                     operator: AssertionOperator,
+                     expected, message=""):
+
+    if isinstance(operator, str):
```

Comment:
Could we get higher readability just by defining enum aliases for all the values that have multiple labels and matching by that below?

Reply:
Enum Aliases are only possible when you define the enum as class, but then the members `==` and `>` are not possible.
So no aliases.

Also we do not need the enum really. it is just needed for future autocompletion.

Reply:
Could one pick `AssertionOperator` from a dictionary to improve readability? Example if one have: 
```
operators = {'contains': AssertionOperator["*="], 'shouldcontain': AssertionOperator["*="] ... }
operator = operators.get(operator)
```
The if madness would be gone and it could be easier to read.